.env.example

```diff
@@ -1,14 +1,11 @@
 # true or false
-FORK=true
```

Comment:
I think we are still using this env variable in the hardhat config file

Reply:
fixed